CHANGES.md

```diff
@@ -48,6 +55,28 @@ interior start and end points of the segments were imported into the database. T
 * Updated citygml4j to 2.10.5.
 * Updated sqlbuilder to 2.2.1.
 
+##### Additions
+
+* The COLLADA/KML/glTF export now supports exporting textures to binary glTF files `.glb`.
+
+* The COLLADA/KML/glTF export now supports exporting textures of `.gltf` files in Draco compression.
+
+##### Changes
+
+* The option to embed textures in `.gltf` files is now set to `true` by default to reduce network calls while visualizing. 
```

Comment:
Moved to branch [glb-draco-support](https://github.com/3dcitydb/importer-exporter/tree/glb-draco-support).